xinference/core/model.py

```diff
@@ -375,7 +375,6 @@ def _wrapper():
 
         return await self._call_wrapper(_wrapper)
 
-    @log_async(logger=logger)
     async def next(
```

Comment:
Why removing this `log_async`? I will remove this `next` in https://github.com/xorbitsai/inference/pull/859. We can keep this decorator in this PR to avoid conflicts.